lib/tests/streamlit/text_input_test.py

```diff
@@ -104,6 +104,24 @@ def test_inside_column(self):
 
         self.assertEqual(text_input_proto.label, "foo")
 
+    def test_autocomplete_defaults(self):
+        """If 'autocomplete' is unspecified, it defaults to the empty string
+        for default inputs, and "new-password" for password inputs.
+        """
+        st.text_input("foo")
+        proto = self.get_delta_from_queue().new_element.text_input
+        self.assertEqual("", proto.autocomplete)
+
+        st.text_input("password", type="password")
```

Comment:
should we split these?

Reply:
In this case, these are the only two defaults we'll ever have, I think - the "proper" way to do this might be to parameter-ize the test, but that'll actually result in more code. So I'd vote for not splitting.